Directory.Build.props

```diff
@@ -2,6 +2,10 @@
   <PropertyGroup>
     <PackageOutputPath>$(MSBuildThisFileDirectory)outputpackages\</PackageOutputPath>
   </PropertyGroup>
+  
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
```

Comment:
>latest [](start = 17, length = 6)

This does not cguarentee consistency between what people have installed and what is on the build servers.  We should pin this to the version we have on the build servers.

Reply:
Spoke w/ Tom in person, and brought him around to using "latest".